esx_service/vmdk_ops.py

```diff
@@ -715,18 +715,33 @@ def handleVmciRequests():
 
     lib.close(sock)  # close listening socket when the loop is over
 
+def usage():
+    print "Usage: %s -p <vSocket Port to listen on>" % sys.argv[0]
 
 def main():
     log_config.configure()
-    logging.info("=== Starting vmdkops service ===")
-
+    logging.info("=== Starting vmdkops service PID====")
     signal.signal(signal.SIGINT, signal_handler_stop)
     signal.signal(signal.SIGTERM, signal_handler_stop)
+    try:
+        port=1019
+        opts, args = getopt.getopt(sys.argv[1:], 'hp:')
```

Comment:
Had a chat with Prashant. A config file might be nice for all things plugin.. have filed https://github.com/vmware/docker-volume-vsphere/issues/439 for the vmci clean up where I will look at the config file needs as well. Currently there is a config file that is only used for logging and also, there is no VMCI module which can be configured on power up that leaves either a global variable for port or change the function call signatures.. For now I am not changing this PR which is needed for take 2 of TP.